lib/src/dartdoc/dartdoc_internals.dart

```diff
@@ -17,6 +17,7 @@ bool isHrefALibrary(String? href) {
   if (href.endsWith('.html')) {
     return false;
   }
-  // libraries have no slash in their hrefs
-  return !href.contains('/');
+  final segments = href.split('/').where((s) => s.isNotEmpty).toList();
```

Comment:
This also removes empty segments in the middle `a//b->a/b` is that ever a problem? should we test for an empty segment only at the end?

Reply:
In this specific case, it would be considered as a bug in dartdoc, however, it doesn't hurt if we remove only the last one. Done.